index.md

```diff
@@ -4,113 +4,117 @@ layout: essential
 class: front-page
 ---
 {% include page_header.html %}
+{% capture newline %}
+{% endcapture %}
 
 {% comment %}
 ##### Content #####
 First we set up & capture the content, then we render it in the scaffolding below.
 {% endcomment %}
 
-
-{% capture intro-left %}
-The [easy-to-use](#easy-to-use),
-[integrated](#integrated),
-[glanceable](#glanceable),
-and [open](#open-ended)
-web-based interface for your servers
-{% endcapture %}
-
-{% capture intro-right %}
-[![Docker screenshot]({{ site.baseurl }}/images/site/screenshot-docker.png)]({{ site.baseurl }}/images/site/screenshot-docker.png){:.screenshot.zoom}
+{% capture intro-lede %}
+The easy-to-use, integrated, glanceable, and open web-based interface for your servers
 {% endcapture %}
 
+{% capture intro-text %}
+## Introducing Cockpit
 
-{% assign blurbs_highlight = 6 %}
+Cockpit is a web-based graphical interface for servers, intended for everyone, especially those who are:
 
-{% capture blurbs %}
+{:.audience-list}
+- **new to Linux** (including Windows admins)
+- **familiar with Linux** and want an easy, graphical way to administer servers
+- **expert admins** who mainly use other tools but want an overview on individual systems
 
-## Easy to use
+Thanks to Cockpit intentionally using system APIs and commands, a whole team of admins can manage a system in the way they prefer, including non-Cockpit commands and utilities right alongside Cockpit.
```

Comment:
This feels a bit odd to me. How about this:
```suggestion
Cockpit uses the official system APIs and commands and does not interfere with other tools. Hence, administrators can manage a system in the way they prefer, including non-Cockpit commands and utilities right alongside Cockpit.
```

Reply:
Thanks for the suggestion! I've been so close to the trees, I haven't seen all of the forest, so to speak.